AutomatedTesting/LuaScripts/ScriptAutomation_sample.lua

```diff
@@ -0,0 +1,8 @@
+LoadLevel("prefabs/default_level.spawnable")
+-- Wait for level to load
+IdleSeconds(5)
+
+StartBudgetTotalCapture()
+IdleFrames(100)
+StopBudgetTotalCapture()
+WritePerfDataToCsvFile("@user@/PerfData/metrics.csv")
```

Comment:
This should probably be moved to the ScriptAutomation Gem's assets directory if this is going to be checked in. I was using this script for testing. Should this be checked in?